e2e/nomostest/predicates.go

```diff
@@ -169,24 +169,33 @@ func HasExactlyLabelKeys(wantKeys ...string) Predicate {
 	}
 }
 
-// HasExactlyImageName ensures a container has the expected image name
-func HasExactlyImageName(containerName string, expectImageName string) Predicate {
+// HasExactlyImage ensures a container has the expected image.
+func HasExactlyImage(containerName string, expectImageName, expectImageTag, expectImageDigest string) Predicate {
```

Comment:
nit: the first `string` type can be omitted as well.

Reply:
done